packages/suite-desktop-core/src/libs/connect-ws.ts

```diff
@@ -210,12 +213,14 @@ export const exposeConnectWs = ({
                     method,
                     payload: rest,
                     origin,
-                    process: {
-                        name: processOnPort.name,
-                        fullPath: processOnPort.fullPath,
-                        warning: !!processOnPort.warning,
-                        icon: await getProcessIcon(processOnPort.fullPath),
-                    },
+                    process: processOnPort
+                        ? {
+                              name: processOnPort.name,
+                              fullPath: processOnPort.fullPath,
+                              warning: !!processOnPort.warning,
+                              icon: await getProcessIcon(processOnPort.fullPath),
+                          }
+                        : undefined,
```

Comment:
wasn't this a 'security' feature? to present users with process ? 🤔

Reply:
Yes, but unfortunately it's not possible to get info on the other process with Flatpak sandboxing.

Reply:
and this is solved in UI in some clever way right? no info -> warning, or something like this.